README_SE.md

```diff
@@ -1,49 +1,43 @@
+Om du kan, använd ditt riktiga namn och lägg till dina projekt eller anknutna organisationer inom parentes.
 
-Klicka sedan **på "Propose new file "** och gå igenom de efterföljande sidorna för att skapa en sammanslagningsbegäran.
+Sedan **klicka på "Propose new file"** (Föreslå ny fil) och gå igenom de efterföljande sidorna för att skapa en begäran om sammanslagning.
 
-Låt oss hålla en fast, men professionell ton.
+Låt oss hålla tonen professionell.
 
-Om du kan, tänk på att dela detta brev på dina forum och sociala medier och meddela journalister som kan vara till hjälp för vår sak.
+Om du kan, dela gärna med dig av detta brev i dina forum eller på sociala medier samt meddela journalister som kan vara till hjälp för vår sak.
 
-Alternativt kan du gaffla och klona repo:n, skapa filen `_data/signed/<användarnamn>.yaml` manuellt, sedan lägga in och skicka in en PR.
+Alternativt kan du skapa en kopia av projektet, skapa filen `_data/signed/<anvandernamn>.yaml` manuellt och sedan "commit" och skicka en PR.
 
-Om du vill stödja brevet **utan att använda Github**, gå hit: https://codeberg.org/rms-support-letter/rms-support-letter/issues/1,
-eller skicka en signerad patch till [signrms@prog.cf](mailto:signrms@prog.cf) eller [~tyil/rms-support@lists.sr.ht](mailto:~tyil/rms-support@lists.sr.ht).
+Om du vill stödja brevet **utan att använda Github**, klicka här: https://codeberg.org/rms-support-letter/rms-support-letter/issues/1,
+eller skicka en signerad "patch" till [signrms@prog.cf](mailto:signrms@prog.cf) eller [~tyil/rms-support@lists.sr.ht](mailto:~tyil/rms-support@lists.sr.ht).
 
-Om du fortfarande behöver hjälp via visuella instruktioner kan du använda [this](https://invidious.snopyta.org/watch?v=1lz5S5oS8CU) video.
+Om du fortfarande behöver hjälp via visuella instruktioner, använd [den här videon](https://invidious.snopyta.org/watch?v=1lz5S5oS8CU).
 
-**Alla signaturer kommer att behandlas inom högst 12 timmar - på grund av en stor mängd PRs**.
+**Alla signaturer kommer att behandlas inom högst 12 timmar - på grund av en hög PR volym**
 
-## Chattrum
+## Chattrum (på engelska)
 
 - **Matrix:** [+free-rms:matrix.org](https://matrix.to/#/+free-rms:matrix.org)
-- **IRC:** #free-rms på [freenode](https://freenode.net)
+- **IRC:** #free-rms on [freenode](https://freenode.net)
 - **Discord:** https://discord.gg/7FWkxG4CsU
-- **Telegram:** https://t.me/free_rms
-
-## Licensiering
-Koden i arkivet är licensierad under GPL-3.0-only.
-
-Bilderna i mappen `assets` är licensierade under [CC BY-SA 3.0](https://creativecommons.org/licenses/by-sa/3.0/legalcode). De sociala förhandsgranskningsbilderna är baserade på https://commons.wikimedia.org/wiki/File:Richard_Matthew_Stallman.jpeg, publicerad under CC BY-SA 3.0, som ursprungligen publicerades som omslaget till O'Reilly-boken `Free as in Freedom: Richard Stallman's Crusade for Free Software by Sam Williams`, publicerad den 1 mars 2002 under GFDL.
-
-Signaturer är inte upphovsrättsligt skyddade.
```

Comment:
Why remove the translation of the licensing notice?

Reply:
The licensing was not mandatory as it was explained somewhere in a discussion thread, so I did the same as I could observe on other translations.

Reply:
It's not mandatory, but it doesn't have to be removed either when it's already there though.

If removing this makes translation easier, then I will accept it.